test/Dialect/Ibis/structure.mlir

```diff
@@ -40,18 +40,18 @@ ibis.class @C {
 // CHECK:           ibis.return [[x]] : ui32
 
 
-// PREP-LABEL: ibis.class @User {
-// PREP:         ibis.instance @c, @C
+// PREP-LABEL: ibis.class @User(%this) {
+// PREP:         [[c:%.+]] = ibis.instance @c, @C
 // PREP:         ibis.method @getAndSetWrapper(%arg: !hw.struct<new_value: ui32>) -> ui32 {
 // PREP:           %new_value = hw.struct_explode %arg : !hw.struct<new_value: ui32>
-// PREP:           %0 = hw.struct_create (%new_value) {sv.namehint = "getAndSet_args_called_from_getAndSetWrapper"} : !hw.struct<x: ui32>
-// PREP:           %1 = ibis.call @c::@getAndSet(%0) : (!hw.struct<x: ui32>) -> ui32
+// PREP:           %1 = hw.struct_create (%new_value) {sv.namehint = "getAndSet_args_called_from_getAndSetWrapper"} : !hw.struct<x: ui32>
```

Comment:
I don't like including namehints in tests but it's a judgement call.

Reply:
I personally like to keep round-trip tests verbatim what the output IR is.